streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -2178,8 +2178,8 @@ void to(final TopicNameExtractor<K, V> topicExtractor,
      * {@link ValueJoiner}, one output for each input {@code KStream} record
      * @see #leftJoin(GlobalKTable, KeyValueMapper, ValueJoiner)
      */
-    <GK, GV, RV> KStream<K, RV> join(final GlobalKTable<GK, GV> globalKTable,
-                                     final KeyValueMapper<? super K, ? super V, ? extends GK> keyValueMapper,
+    <GK, GV, RV> KStream<K, RV> join(final GlobalKTable<GK, GV> globalTable,
+                                     final KeyValueMapper<? super K, ? super V, ? extends GK> keySelector,
```

Comment:
renamed `globalKTable` to `globalTable`
renamed `keyValueMapper ` to `keySelector`